client/my-sites/post-relative-time-status/index.jsx

```diff
@@ -96,10 +97,22 @@ class PostRelativeTime extends React.PureComponent {
 			statusText = this.props.translate( 'pending review' );
 			statusClassName += ' is-pending';
 		} else if ( status === 'future' ) {
-			const scheduledTime = this.props.moment( this.props.post.date ).fromNow();
-
-			statusText = this.props.translate( 'scheduled %(scheduledTime)s', {
-				comment: '%(scheduledTime)s is a future human time, for example "in 3 days"',
+			const moment = this.props.moment;
+			const scheduledDate = moment( this.props.post.date );
+			const now = moment();
+			const scheduledTime =
+				scheduledDate.diff( now, 'days' ) <= 7
+					? scheduledDate.calendar()
+					: scheduledDate.format( 'LL' ) +
+					  ' ' +
+					  this.props.translate( 'at', {
+							context: 'scheduled for 21 April at 02:00',
+					  } ) +
+					  ' ' +
+					  scheduledDate.format( 'LT' );
```

Comment:
re https://github.com/Automattic/wp-calypso/pull/40843/files#r404814741:
```suggestion
			const scheduledTime = scheduledDate.calendar( null, {
				sameElse: this.props.translate( 'LL [at] LT', {
					comment: 'moment.js formatting string'
				}
			);
```

Reply:
Thank you, fixed! :)